docs/getting-started/get-started-nuxt-3.md

```diff
@@ -8,170 +8,100 @@ heading: Get Started
-```
+## Install the Data Grid Component
 
-With the above, we not only add the packages of the `Grid` and `DropDownList` but also add another important package – `kendo-data-query`. It contains useful functions for client-side data operations.
+Kendo UI for Vue is distributed as multiple NPM packages, scoped to `@progress`. For example, the name of the Grid package is `@progress/kendo-vue-grid`. To use the Grid in your app, add the component and its [dependencies](https://www.telerik.com/kendo-vue-ui/components/grid/#toc-dependencies):
 
-To install the Window component run the following:
 ```sh
-npm install --save @progress/kendo-vue-dialogs @progress/kendo-licensing @progress/kendo-svg-icons
-```
-
-> To demonstrate the features of the Nuxt Framework, we will separate the implementation of the Grid, DropDownList, and Window components into separate Vue components that pass parameters to each other. 
-
-## 5. Import the Kendo UI for Vue CSS styles
-
-Kendo UI for Vue includes four gorgeous themes, which are all available as separate NPM packages. The available theme packages are [@progress/kendo-theme-default](https://www.npmjs.com/package/@progress/kendo-theme-default), [@progress/kendo-theme-bootstrap](https://www.npmjs.com/package/@progress/kendo-theme-bootstrap), [@progress/kendo-theme-material](https://www.npmjs.com/package/@progress/kendo-theme-material) and [@progress/kendo-theme-fluent](https://www.npmjs.com/package/@progress/kendo-theme-fluent).
-
-Let’s take the Default theme and install it just like we did with the component packages:
-```sh
-npm install --save @progress/kendo-theme-default
-```
-
-To add the CSS definitions from the theme package in our project, we need to include them in the `nuxt.config.ts` file as follows:
-
-```js
-export default defineNuxtConfig({
-    css: [
-        '@progress/kendo-theme-default/dist/all.css',
-    ],
-})
+npm install --save @progress/kendo-vue-grid @progress/kendo-data-query @progress/kendo-licensing @progress/kendo-vue-animation @progress/kendo-vue-data-tools @progress/kendo-vue-dateinputs @progress/kendo-vue-dropdowns @progress/kendo-vue-inputs @progress/kendo-vue-indicators @progress/kendo-vue-intl @progress/kendo-vue-popup
 ```
-
-## 6. Add a Kendo UI for Vue DropDownList
-Now that you have everything set up and ready to go, let’s begin using the Kendo UI for Vue components, starting with the [DropDownList](slug:overview_dropdownlist) component.
-
-As mentioned above, each of the Kendo UI for Vue components we are going to use in the getting started example will be added to a separate Vue component. To create the file in which we will add the DropDownList's logic run the following command:
-
+<!---
```

Comment:
Yes, the hiding is intentional, because they must be enabled once the new docs/demos platform is available as it will support tabbed code snippets. Currently, they add unnecessary length to the article.